spec/controllers/api/emoji_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Api::EmojiController, type: :controller do
+  # :FIXME APIのテストなのでrequest specが適当らしい。誰か書きなおしてください
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)